colossalai/fx/profiler/__init__.py

```diff
@@ -1,3 +1,9 @@
+try:
+    from ._meta_registrations import *
+except:
+    import torch
+    print(f'_meta_registrations seems to be incompatible with PyTorch {torch.__version__}.')
+from .meta_tensor import *
 from .registry import *
 from .profiler_function import *
 from .profiler_module import *
```

Comment:
Do not use `*` in import as it might mess up the import namespace. You should follow the steps below.
1. give concrete name for import, i.e. `from a import b` instead of `from a import *`
2. add `__all__` for namespace constraints, e.g. `__all__ = ['b']`